code/graphics/debug_sphere.cpp

```diff
@@ -0,0 +1,38 @@
+#include "debug_sphere.h"
+
+#include "globalincs/vmallocator.h"
+
+#include "render/3d.h"
+
+SCP_vector<debug_sphere> debug_sphere::Spheres;
+
+debug_sphere::debug_sphere(const vec3d& pos_in, float rad_in)
+{
+	pos = pos_in;
+	radius = rad_in;
+}
+void debug_sphere::add(const vec3d& pos, float rad)
+{
+	debug_sphere::Spheres.emplace_back(debug_sphere(pos, rad));
```

Comment:
I *think* you can take `debug_sphere` out of this line and it will be constructed in-place.

Reply:
@Goober5000 I wasn't sure at first which instance `debug_sphere` you meant so I hit them both :)

Tested afterwards and all works as it should

Reply:
I meant the second one, but you're right, the first one is removable too. 👍